provider/export/src/export_impl.rs

```diff
@@ -141,8 +141,8 @@ impl ExportDriver {
             let instant1 = Instant::now();
 
             if marker.is_singleton {
-                if provider.iter_ids_for_marker(marker)? != HashSet::from_iter([Default::default()])
-                {
+                let supported = provider.iter_ids_for_marker(marker)?;
+                if supported.len() != 1 || !supported.first().unwrap().is_default() {
```

Comment:
Nit: Write this in a way that avoids the unwrap. Something like

```rust
if !matches!((iter.next(), iter.next()), (Some(DataIdentifier::DEFAULT), None)) { ... }
```

Reply:
^ This wasn't resolved, either

Reply:
My suggestion is to remove the unwrap. I offered a one-liner to do so. I would normally block a PR on this but since it is in the `export` code I won't block on it.